apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js

```diff
@@ -81,6 +86,7 @@ function HelpCenterContent() {
 				</>
 			) }
 			<HelpCenter handleClose={ closeCallback } />
+			<WhatsNewGuide onClose={ handleCloseWhatsNew } />
```

Comment:
Why are we adding this to the help center as opposed to somewhere else? (Calypso proper, adjusting the whats-new etk plugin, etc.).  

In general, the help center already adds and controls an instance of the `WhatNewGuide` at [a deeper level](https://github.com/Automattic/wp-calypso/blob/ea8ff3cc00e4cf6eea076d7afea7e71ff898bb35/packages/help-center/src/components/help-center-more-resources.tsx#L154). It doesn't seem like it should manage two separate instances of this guide at two different levels. And it sounds like our goal here is to make opening/managing this independent of the help center being open, this signals to me this instance of the WhatsNewGuide shouldn't be in the help center in the first place.

With whats-new as a package, [a separate ETK plugin](https://github.com/Automattic/wp-calypso/blob/ea8ff3cc00e4cf6eea076d7afea7e71ff898bb35/apps/editing-toolkit/editing-toolkit-plugin/whats-new/src/whats-new.js#L43) as well, and our desire to be independent of help center, it seems like we could and should find this logic a better home and corresponding data store?

Reply:
Good question! that's something I spend a lot of time working out.

I'm actually adding it again here because I'm taking it out of the [help-center package](https://github.com/Automattic/wp-calypso/pull/87875/files#diff-b3c3c91e9f478e4470a4a2106b69ddbfa8028619b1331c6b195361419de1c6cbL154)!

I moved it to by [loaded in calypso by the main layout](https://github.com/Automattic/wp-calypso/pull/87875/files#diff-7d1eb73ef67c978553b9d47cf17a8f4b9d93bd396ee6cb5ce89e75aedb8b0f19R130). And use the redux action setShowWhatsNewModal() to open it either from the [main layout on load](https://github.com/Automattic/wp-calypso/pull/87875/files#diff-7d1eb73ef67c978553b9d47cf17a8f4b9d93bd396ee6cb5ce89e75aedb8b0f19R117) or from the help-center package.

The problem is that the code running in the ETK does not share the redux store, so I need to ensure an instance of the whats-new modal is also loaded in the ETK's frame. 

So I need to load it in the ETK code, but not in a package that is shared with calypso, or else it would be loaded twice.

I also first thought to put it in the `whats-new` ETK package, but I found that this package doesn't actually seem to be active in the editor currently. It appears that the intent of the `whats-new` package is to add another menu item to the tools menu. It[ registers a javascript plugin called 'whats-new' ](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/whats-new/src/whats-new.js#L43C1-L43C15) but the plugin is currently never loaded.

It is also good that since the `whats-new` modal is used by the etk help center package, this package is responsible for loading it. So that way the package loads its own dependencies.

After this PR, I will look into deleting/migrating that whats-new ETK package because it seems to be unused, I asked in the Vertex slack if they have any more context on it. p1709169478558419-slack-C02T4NVL4JJ

Reply:
Ohh, that makes a bit more sense. Thanks for explaining 😄